nrf52-hal-common/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+authors = [
+    "James Munns <james.munns@gmail.com>",
+    "Hanno Braun <hanno@braun-robotics.com"
+]
+categories = ["embedded", "hardware-support", "no-std"]
+description = "HAL for the nRF52 family of microcontrollers"
```

Comment:
If I understand correctly, this crate is intended to be a dependency for HAL implementations, not as something to be used by HAL users directly. Unless this is incorrect, this description should be updated to reflect this.

Reply:
I realized that I worded my criticism too strongly. It makes perfect sense to depend on the common crate whenever you can get away with it (for example in libraries). Still, the description should be updated to state that this is a common crate for the whole nRF52 series, and that there are more specific crates to be found.

Reply:
Sounds reasonable to me; amended in the latest commit